package.json

```diff
@@ -134,16 +131,9 @@
     "webpack": "^5.77.0"
   },
   "resolutions": {
-    "@types/react": "^18.0.32",
-    "commitizen": "^4.3.0"
```

Comment:
We don't want this for anyone who wants to use it? :') I guess it's only me who would miss it (although I don't need it haha).

Reply:
Haha, time to rely on our team's natural discipline